### DISCUSSION TITLE
Uno calculator on Mac - no keyboard support

### DISCUSSION THREAD

**Discussion by Eyal-Lasko-QM** [unoplatform/uno#11678] [Q&A]
Hi community,

Recently moved from Windows to Mac and found the Uno calculator.
It works fine online, but I can't use the keyboard for basic input when using the 'standalone' version on Mac (M1 Max chip, OS 13).

Is it me or just a feature left outside ??? 
Thanks

**Comment by jeromelaban** [comment]
Thanks for the report! We have an update in the works that will bring keyboard support, by means of using mac Catalyst.

**Comment by Eyal-Lasko-QM** [comment]
THANKS for the quickest-ever reply!

any estimation of when this update will be released?
will the standalone Mac version auto-update or should I check the website periodaically?

**Reply by jeromelaban** [reply]
There's no estimate for an update, but it will auto update if you installed from the mac app store.
